packages/providers/src.ts/base-provider.ts

```diff
@@ -186,9 +195,22 @@ export interface Avatar {
 }
 
 export interface ProviderOptions {
-    headers?: Record<string, string>
+    headers: Record<string, string>
+    retry: RetryOptions
 }
 
+export interface RetryOptions {
```

Comment:
Do we need to export it? As far as I see we are not using it in another package, so exporting is not mandatory.